src/image-sync-manager.ts

```diff
@@ -1,7 +1,6 @@
 import {ImagePlayer} from './image-player';
 
-export const TIMED_METADATA_ADDED = 'timedmetadataadded'; // TODO: should be enum from cuePointManager
-export const SLIDE_CUE_POINT_TYPE = 'slides'; // TODO: 'slides' should be enum from cue-point plugin
+const ThumbCuePointType = 'thumbCuePoint.Thumb';
```

Comment:
could be enum from cue-point plugin?

Reply:
add TODO: and once cue-point service deployed - use enum from there